framework/src/main/java/org/checkerframework/common/returnsreceiver/messages.properties

```diff
@@ -0,0 +1 @@
+invalid.this.location=invalid location for @This annotation
```

Comment:
You could either use the existing `type.invalid` annotation or follow that naming `type.invalid.this.location`.